### PR TITLE
Postgres

### PR DESCRIPTION
**Pull request by jawad04**

### CODE REVIEW

**Comment by jawad04** [comment]
@marymatt504, Can you please take a look at my seed.js and schema.sql files please?

File: SQL/README.md [hunk 1]
```diff
@@ -31,6 +27,17 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
 - Node 6.13.0
 - etc
 
+
+## Endpoints
+
+| Action        | Request       | Route            |  Purpose  |
+| ------------- | -------------| ----------------| ---------------|
```
Comment (marymatt504) [inline SQL/README.md:34]: could add spacing to align table

File: SQL/database/connection.js [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+const connection = pg.Pool({
+  // user: 'root'
+  user: 'postgres',
   database: 'fandango',
-  host: process.env.RDS_HOSTNAME,
-  user: process.env.RDS_USERNAME,
-  password: process.env.RDS_PASSWORD,
-  port: process.env.RDS_PORT
+  host: '127.0.0.1',
+  user: 'root',
```
Comment (marymatt504) [inline SQL/database/connection.js]: is user in here twice?

**Review by marymatt504 — APPROVED** [review]
great work, Jawad!

File: SQL/database/connection.js (hunk 2 above)
Comment (jawad04) [reply SQL/database/connection.js]: Oh, great catch. I will fix it. Thanks :)

**Pull request merged**